run_dir/static/js/index.js

```diff
@@ -118,7 +118,103 @@ $('body').on('click', '.group', function(event) {
   $($("#prio_projs_table").DataTable().column(0).header()).trigger("click")
 });
 
+function plot_sum_data(){
+  $.getJSON("/api/v1/sensorpush", function(data) {
+    var frig_series = [];
+    var freez_series = [];
+    $.each(data, function(id, sensordata){
+      var timedata = sensordata.samples;
+      var sensname = sensordata.sensor_name;
+      var sercolor = '';
+      for (i in timedata) {
+        timedata[i][0] = Date.parse(timedata[i][0]);
+        date = new Date()
+        date.setTime(timedata[i][0])
```

Comment:
I can't see where this variable is used?

Reply:
I think that's just how you set the time in JS? 
I copied the logic from here: https://github.com/SciLifeLab/genomics-status/blob/master/run_dir/static/js/sensorpush.js#L3

But I could be wrong 🤔

Reply:
It is yeah, but you are setting it in a variable called `date` which is never used.

Reply:
You're right!